doc/base/classes.xml

```diff
@@ -695,6 +702,8 @@
 			</argument>
 			<description>
 				Return a weak reference to an object.
+
```

Comment:
Same here, the empty line should be removed, each line will already be treated as a separate paragraph.